apps/web/src/components/templates/email-editor/EmailContentCard.tsx

```diff
@@ -92,11 +88,7 @@ export function EmailContentCard({
       {!isIntegrationActive ? (
         <LackIntegrationError
           channelType="E-Mail"
-          text={
-            enabled
-              ? 'Looks like you haven’t configured your E-Mail provider yet, visit the integrations page to configure.'
-              : undefined
-          }
+          text={'Looks like you haven’t configured your E-Mail provider yet, visit the integrations page to configure.'}
```

Comment:
this is a bit problematic for the self hosted version because the other text gives your more info if it is self hosted then this one does